terraform/db.tf

```diff
@@ -0,0 +1,39 @@
+
+resource "aws_db_subnet_group" "pgsql" {
+  name       = "pgsql"
+  subnet_ids = [aws_subnet.private_az1.id, aws_subnet.private_az2.id]
+}
+
+resource "aws_rds_cluster_instance" "pgsql" {
+  count              = var.target_env == "prod" ? 2 : 1
+  identifier         = "${local.db_name}-${count.index}"
+  cluster_identifier = aws_rds_cluster.pgsql.id
+  instance_class     = "db.r5.large"
+  engine             = aws_rds_cluster.pgsql.engine
+  engine_version     = aws_rds_cluster.pgsql.engine_version
+}
+
+resource "aws_rds_cluster" "pgsql" {
+  cluster_identifier  = local.db_name
+  engine              = "aurora-postgresql"
```

Comment:
I wonder what the difference in postgres and aurora postgres engine would be? 
Docs from aws say just postgres is also an option, tf docs says aurora.. Unsure about this. 

would be good to know the difference. 

https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-cluster.html